Automattic/wp-calypso
# Adds Anti-Spam icon to ProductIcon component package

#### Changes proposed in this Pull Request

* Adds a new Jetpack Anti-Spam product icon. Note this is a placeholder while we finalize details.

#### Testing instructions



* On Calypso, go to `/devdocs/design/product-icon`.
* Verify icon is there and looks correct.

#### Screenshot
<img width="250" alt="Screen Shot 2020-07-06 at 1 12 23 PM" src="https://user-images.githubusercontent.com/1760168/86620460-57580600-bf8a-11ea-9898-7e9307cce6d4.png">


Fixes 1181531115069428-as-1182646983386243
